src/pages/Blog/index.tsx

```diff
@@ -0,0 +1,399 @@
+import React, { useEffect, useState } from "react";
+import useStyles from "./styles";
+import { useTranslation } from "react-i18next";
+import {
+  Avatar,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  CardMedia,
+  Container,
+  createStyles,
+  Grid,
+  Paper,
+  Tab,
+  Tabs,
+  Theme,
+  Typography,
+  withStyles,
+} from "@material-ui/core";
+import Footer from "../../components/Footer";
+import ReactMarkdown from "react-markdown";
+import { BLOG_KEYWORDS, VIEW_PORT } from "../../constants";
+import Sponsor from "../../components/Sponsor";
+import CustomTag from "../../components/CustomTag";
+import Pagination from "@material-ui/lab/Pagination";
+import blogs from "../../posts.json";
+
+interface StyledTabProps {
+  label: string;
+  value: string;
+}
+
+interface TabProps {
+  id: string;
+  title: string;
+  blog: string;
+  description: string;
+  image: string;
+  tags: string;
+  author: string;
+  length: number;
+}
+
+const Blog: React.FC = () => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const [jsonMdData, setJsonMdData] = useState<any>();
```

Comment:
Please initialize with ''. Else if may lead to undefined.

Reply:
done